// @esri/calcite-components/dist/cjs/bs-90a14199.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.10.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-e62854f3.js');
require('./index-631c1935.js');
require('./form-00b7de4b.js');
require('./dom-6172ed4c.js');
require('./guid-67658644.js');
require('./resources-01ae6233.js');
require('./interactive-e0c7a04c.js');
require('./browser-bc2f2efb.js');
require('./key-1b72be47.js');
require('./label-40e4cd77.js');
require('./loadable-77539c62.js');
require('./locale-786ce6b3.js');
require('./observers-c825c70d.js');
require('./focusTrapComponent-014b315b.js');
require('./math-b503dee6.js');
require('./t9n-5def0889.js');
require('./openCloseComponent-7235170a.js');
require('./component-75681f41.js');

// Bosnian [bs]
var locale = {
  name: 'bs',
  weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
  months: 'januar_februar_mart_april_maj_juni_juli_august_septembar_oktobar_novembar_decembar'.split('_'),
  weekStart: 1,
  weekdaysShort: 'ned._pon._uto._sri._čet._pet._sub.'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj._jun._jul._aug._sep._okt._nov._dec.'.split('_'),
  weekdaysMin: 'ne_po_ut_sr_če_pe_su'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY H:mm',
    LLLL: 'dddd, D. MMMM YYYY H:mm'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports.default = locale;
