// @esri/calcite-components/dist/esm/en-gb-8144d60e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.10.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-b4438756.js';
import './index-89cd0584.js';
import './form-093f5b04.js';
import './dom-9ac56900.js';
import './guid-805a73d1.js';
import './resources-6ed27057.js';
import './interactive-2b2de863.js';
import './browser-6b1c93d7.js';
import './key-f3638642.js';
import './label-0873b8dd.js';
import './loadable-1917c524.js';
import './locale-07424fa5.js';
import './observers-469c8c46.js';
import './focusTrapComponent-3b9fb942.js';
import './math-84ae8cd7.js';
import './t9n-0cedcbfc.js';
import './openCloseComponent-baf628d7.js';
import './component-d6d13dbf.js';

// English (United Kingdom) [en-gb]
var locale = {
  name: 'en-gb',
  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
  weekdaysShort: 'Sun_Mon_Tue_Wed_Thu_Fri_Sat'.split('_'),
  weekdaysMin: 'Su_Mo_Tu_We_Th_Fr_Sa'.split('_'),
  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split('_'),
  monthsShort: 'Jan_Feb_Mar_Apr_May_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
  weekStart: 1,
  yearStart: 4,
  relativeTime: {
    future: 'in %s',
    past: '%s ago',
    s: 'a few seconds',
    m: 'a minute',
    mm: '%d minutes',
    h: 'an hour',
    hh: '%d hours',
    d: 'a day',
    dd: '%d days',
    M: 'a month',
    MM: '%d months',
    y: 'a year',
    yy: '%d years'
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd, D MMMM YYYY HH:mm'
  },
  ordinal: function ordinal(n) {
    var s = ['th', 'st', 'nd', 'rd'];
    var v = n % 100;
    return "[" + n + (s[(v - 20) % 10] || s[v] || s[0]) + "]";
  }
};
dayjs.locale(locale, null, true);

export default locale;
