// @esri/calcite-components/dist/esm/ja-dcc50b14.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.10.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-b4438756.js';
import './index-89cd0584.js';
import './form-093f5b04.js';
import './dom-9ac56900.js';
import './guid-805a73d1.js';
import './resources-6ed27057.js';
import './interactive-2b2de863.js';
import './browser-6b1c93d7.js';
import './key-f3638642.js';
import './label-0873b8dd.js';
import './loadable-1917c524.js';
import './locale-07424fa5.js';
import './observers-469c8c46.js';
import './focusTrapComponent-3b9fb942.js';
import './math-84ae8cd7.js';
import './t9n-0cedcbfc.js';
import './openCloseComponent-baf628d7.js';
import './component-d6d13dbf.js';

// Japanese [ja]
var locale = {
  name: 'ja',
  weekdays: '日曜日_月曜日_火曜日_水曜日_木曜日_金曜日_土曜日'.split('_'),
  weekdaysShort: '日_月_火_水_木_金_土'.split('_'),
  weekdaysMin: '日_月_火_水_木_金_土'.split('_'),
  months: '1月_2月_3月_4月_5月_6月_7月_8月_9月_10月_11月_12月'.split('_'),
  monthsShort: '1月_2月_3月_4月_5月_6月_7月_8月_9月_10月_11月_12月'.split('_'),
  ordinal: function ordinal(n) {
    return n + "\u65E5";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'YYYY/MM/DD',
    LL: 'YYYY年M月D日',
    LLL: 'YYYY年M月D日 HH:mm',
    LLLL: 'YYYY年M月D日 dddd HH:mm',
    l: 'YYYY/MM/DD',
    ll: 'YYYY年M月D日',
    lll: 'YYYY年M月D日 HH:mm',
    llll: 'YYYY年M月D日(ddd) HH:mm'
  },
  meridiem: function meridiem(hour) {
    return hour < 12 ? '午前' : '午後';
  },
  relativeTime: {
    future: '%s後',
    past: '%s前',
    s: '数秒',
    m: '1分',
    mm: '%d分',
    h: '1時間',
    hh: '%d時間',
    d: '1日',
    dd: '%d日',
    M: '1ヶ月',
    MM: '%dヶ月',
    y: '1年',
    yy: '%d年'
  }
};
dayjs.locale(locale, null, true);

export default locale;
