// @esri/calcite-components/dist/esm/nl-510ce2f9.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.10.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-b4438756.js';
import './index-89cd0584.js';
import './form-093f5b04.js';
import './dom-9ac56900.js';
import './guid-805a73d1.js';
import './resources-6ed27057.js';
import './interactive-2b2de863.js';
import './browser-6b1c93d7.js';
import './key-f3638642.js';
import './label-0873b8dd.js';
import './loadable-1917c524.js';
import './locale-07424fa5.js';
import './observers-469c8c46.js';
import './focusTrapComponent-3b9fb942.js';
import './math-84ae8cd7.js';
import './t9n-0cedcbfc.js';
import './openCloseComponent-baf628d7.js';
import './component-d6d13dbf.js';

// Dutch [nl]
var locale = {
  name: 'nl',
  weekdays: 'zondag_maandag_dinsdag_woensdag_donderdag_vrijdag_zaterdag'.split('_'),
  weekdaysShort: 'zo._ma._di._wo._do._vr._za.'.split('_'),
  weekdaysMin: 'zo_ma_di_wo_do_vr_za'.split('_'),
  months: 'januari_februari_maart_april_mei_juni_juli_augustus_september_oktober_november_december'.split('_'),
  monthsShort: 'jan_feb_mrt_apr_mei_jun_jul_aug_sep_okt_nov_dec'.split('_'),
  ordinal: function ordinal(n) {
    return "[" + n + (n === 1 || n === 8 || n >= 20 ? 'ste' : 'de') + "]";
  },
  weekStart: 1,
  yearStart: 4,
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD-MM-YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'over %s',
    past: '%s geleden',
    s: 'een paar seconden',
    m: 'een minuut',
    mm: '%d minuten',
    h: 'een uur',
    hh: '%d uur',
    d: 'een dag',
    dd: '%d dagen',
    M: 'een maand',
    MM: '%d maanden',
    y: 'een jaar',
    yy: '%d jaar'
  }
};
dayjs.locale(locale, null, true);

export default locale;
